src/proto/grpc/testing/control.proto

```diff
@@ -37,17 +37,32 @@ package grpc.testing;
 enum ClientType {
   SYNC_CLIENT = 0;
   ASYNC_CLIENT = 1;
+  CLIENT_VARIANT_1 = 2;
```

Comment:
Rather than saying "latency" or "throughput," I propose that we let this proto specify a "reasonable upper bound" on the number of channels and # of outstanding RPCs for each config supported. A pure latency play would set those both to 1, a pure throughput play would leave those unlimited. But the reality is that heavily multithreaded sync implementations are going to have limits in reality, and this would be a way to specify those explicitly.